packages/circuit-ui/components/NotificationBanner/NotificationBanner.module.css

```diff
@@ -10,9 +9,16 @@
     visibility 200ms ease-in-out;
 }
 
-@media (min-width: 960px) {
-  .base {
+.grid {
+  display: grid;
+  grid-template-areas: "content image" "button image";
+  grid-template-columns: minmax(150px, 1fr) auto;
+}
+
+@container cui-notification-banner (width > 960px) {
+  .grid {
     grid-template-areas: "image content button";
+    grid-template-columns: auto minmax(150px, 1fr) auto;
```

Comment:
This ensures proper alignment of the columns when there's more space available than necessary.